pages/[username]/activity.tsx

```diff
@@ -47,7 +47,7 @@ export default function UserFeed({ username, eventId }: UserActivityProps) {
 
   return (
     <GalleryRoute
-      navbar={<GalleryNavbar username={username} queryRef={query} />}
+      navbar={<GalleryNavbar galleryRef={null} username={username} queryRef={query} />}
```

Comment:
anywhere we're passing `galleryRef={null}` it means there is no gallery in context here